main/Settings/Private/settingsDefaultLoader.cpp

```
@@ -0,0 +1,77 @@
+#include <nvs.h>
+
+#include <string.h>
+
+#include "settingsLoader.h"
+#include "../settingsManager.h"
+
+static const char *TAG = "settings_default_loader";
+
+void SettingsManager_InitDefaults(SettingsManager_State_t *p_State)
+{
+    memset(&p_State->Settings, 0, sizeof(App_Settings_t));
```

Cppcheck (2.20.0)

[error] Using 'memset' on struct that contains a 'std

(memsetClass)